packages/sdk/src/client.ts

```diff
@@ -403,8 +403,11 @@ export function SoundClient({
     }
 
     let flags = 0
-    if (editionConfig.shouldFreezeMetadata) flags |= 1
-    if (editionConfig.shouldEnableMintRandomness) flags |= 2
+    if (editionConfig.shouldFreezeMetadata) flags |= editionInitFlags.METADATA_IS_FROZEN
+    if (editionConfig.shouldEnableMintRandomness) flags |= editionInitFlags.MINT_RANDOMNESS_ENABLED
+
+    // Automatically enable Opensea OperatorFilter for royalties
+    flags |= editionInitFlags.OPERATOR_FILTERING_ENABLED
```

Comment:
this should be specified in the editionConfig, we can set default values, but we should let other users opt out of this jank